### PR TITLE
Upstream pull of Credits Page for Game

### PR DESCRIPTION
**Pull request by andywales83**
Credits Page - The End!!!

This page is the ending page of the game with the developers credited , including links to their repositories.

The page follows the hierarchy of 

* Main Title -  Needs More Cowbells
* Sub Title - The End!!
* Credits Section - Credits the developers and links to their repositories
* Main Game Repository Section - link to the main repository for the game.
* Start Again Button - Button can be linked to the main page of the game.

Start button has been tested and currently works against index.html.
All repository links work and are set to open the repository of the individual in a new window

### CODE REVIEW

**Review by CraigThomasson — COMMENTED** [review]
LGTM. going to wait for Anthony to say before we merge.

**Review by auxfuse — APPROVED** [review]
LGTM 🎉 🎉

**Pull request merged**
